Pipfile

```diff
@@ -13,6 +13,7 @@ flask-cors = "*"
 flask-httpauth = "*"
 gunicorn = "*"
 gevent = {version = "*", platform_python_implementation="=='CPython'"}
+google-cloud-pubsub = "*"
```

Comment:
should we use a version for this?

Reply:
The version is locked in the Pipfile.lock.  Pipenv, by design, encourages you do leave things as * so when you regenerate the lockfile, you get minor package updates for 'free' (it's free until something breaks, but that's not as often as you'd think)